config/iroha_test_config.json

```diff
@@ -47,7 +47,7 @@
     "FETCH_SIZE": 10
   },
   "BLOCK_SYNC": {
-    "GOSSIP_PERIOD_MS": 10000,
+    "GOSSIP_PERIOD_MS": 1000,
```

Comment:
This parameter should also be updated [here](https://github.com/hyperledger/iroha/blob/iroha2-dev/cli/README.md?plain=1#L86) I guess